hack/sast.sh

```diff
@@ -6,17 +6,24 @@
 
 set -e
 
-root_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"
+report_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"
```

Comment:
Another option is to get the dir to the repository instead of the dir where the file located. 

```bash
root_dir="$(git rev-parse --show-toplevel)"
```